app/assets/stylesheets/styles.css

```diff
@@ -0,0 +1,91 @@
+@import url('https://fonts.googleapis.com/css2?family=Montserrat:ital,wght@0,100;0,200;0,400;0,700;1,100;1,200;1,300&display=swap');
+@import url('https://fonts.googleapis.com/css2?family=Roboto:wght@300;400;700&display=swap');
+
+body {
+  margin: 0;
+  padding: 0;
+  box-sizing: border-box;
+}
+
+main {
+  width: 320px;
+  height: 100vh;
+  background-color: rgb(241, 241, 241);
+}
+
+.logo {
+  position: relative;
+  margin: 0 auto;
+  text-align: center;
+  width: 50%;
+  top: 40vh;
+  font-family: Arial, Helvetica, sans-serif;
+  font-size: 40px;
+}
+
+.login_signup {
+  display: flex;
+  flex-direction: column;
+  width: 100%;
+  margin: auto;
+  margin-top: 60vh;
+  gap: 25px;
+}
+
+.session {
+  width: 90%;
+  padding: 15px 0;
+  margin: auto;
+  background-color: #3778c2;
+  text-align: center;
+}
+
+.register {
+  background-color: white;
+}
+
+.session_text_login {
+  text-decoration: none;
+  font-size: 17px;
+  font-weight: bold;
+  color: white;
+  font-family: 'Roboto', sans-serif;
+}
+
+.session_text_signup {
+  text-decoration: none;
+  font-size: 17px;
+  font-weight: bold;
+  color: #434b54;
+  font-family: 'Roboto', sans-serif;
+}
+
+.category-image {
+  height: 50px;
+  width: 50px;
+}
+
+.details_of_user {
+  display: flex;
+}
+
+.username {
+  font-size: 14px;
+  text-align: center;
+}
+
+.cat_img div {
+  display: flex;
+  width: 100%;
+  justify-content: space-between;
+}
+
+@media (min-width: 320px) and (max-width: 426px) {
```

Comment:
- [ ] In all the projects in Microverse, we consider the devices under 768px width mobile devices. so kindly consider changing your `max-width` to `768px` so that your app works fine and takes all the screen width on devices having upto 768px width.